packages/example-react/.storybook/main.js

```diff
@@ -12,6 +12,9 @@ module.exports = {
     core: {
         builder: 'storybook-builder-vite',
     },
+    features: {
+      storyStoreV7: true,
```

Comment:
@joshwooding would you mind trying to enable addon-a11y?  I'm seeing an error in my own react app when I try to enable storyStoreV7 using that addon in recent versions of storybook.  (the addon has been working fine previously, and enabling the story store v7 did too, or at least it wasn't breaking anything in older beta versions, but does now.)

Reply:
Hmm, everything is working for me @IanVS What kind of behaviour are you seeing?

Reply:
Ok cool.  Maybe some of the changes you've made here in this PR fix it.  I just thought it'd be worth checking out.  We should uncomment the line to make sure it keeps working.  

In my own app, I was seeing this at runtime:

```
index.js:30 Uncaught Error: Accessing non-existent addons channel, see https://storybook.js.org/basics/faq/#why-is-there-no-addons-channel
    at AddonStore2.getChannel (index.js:30)
    at a11yRunner.js? [sm]:19
```

But, that wasn't using this branch.

Reply:
I think uncommenting is a good idea :)